docs/send-data/kubernetes/collecting-logs.md

```diff
@@ -504,6 +504,128 @@ sumologic:
             manual_prefix_regexp: (\\{\".*|\\[?\\d{4}-\\d{1,2}-\\d{1,2}.\\d{2}:\\d{2}:\\d{2}.*)
 ```
 
+## Sending data to multiple targets
```

Comment:
We should also leave a note specifying this is available from v4.8.

Reply:
Added